libs/api-catalogue/services/src/lib/api-catalogue-services.ts

```diff
@@ -0,0 +1,3 @@
+export function apiCatalogueServices(): string {
```

Comment:
Looks like this function isn't used, but was gonna comment on the name.
`getApiCatalogueServices` > `apiCatalogueServices`
(or change this to a constant)

Reply:
Removed, was just autogenerated with new lib and forgot to remove